tests/unit/developers-serializer.test.js

```diff
@@ -0,0 +1,54 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiSubset = require('chai-subset');
+const _ = require('lodash');
+
+const developersSerializer = appRoot.require('api/v1/serializers/developers-serializer');
+const { getDefinitionProps, testSingleResource, testMultipleResources } = appRoot.require('tests/unit/serializer-test-helpers.js');
+const testData = require('./test-data');
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.use(chaiSubset);
+const { expect } = chai;
+
+describe('Test developers-serializer', () => {
+  const { rawDevelopers } = testData;
+  const resourceType = 'developer';
+
+  it('serializeDeveloper should form a single JSON result as defined in openapi', () => {
+    const { serializeDeveloper } = developersSerializer;
+
+    const serializedDeveloper = serializeDeveloper(rawDevelopers[0]);
+    testSingleResource(serializedDeveloper, resourceType, _.omit(rawDevelopers[0], ['id']));
+  });
+
+  it('serializeDevelopers should form a multiple JSON result as defined in openapi', () => {
+    const { serializeDevelopers } = developersSerializer;
+
+    const serializedDevelopers = serializeDevelopers(rawDevelopers, testData.paginationQueries);
+    testMultipleResources(serializedDevelopers);
+
+    expect(serializedDevelopers).to.have.all.keys(_.keys(getDefinitionProps('DeveloperResults')));
+  });
+
+  it(`serializeDevelopers should be rejected when
+        1. data is passed without queries
```

Comment:
When you have two different test cases like this, it would probably be better to actually generate two different test cases. This way, you'll know which case failed in the output. Could you fix this here and in the other situations where this occurs?